Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -1134,6 +1134,67 @@ def addCKFTracks(
     return s
 
 
+def addGx2fTracks(
+    s: acts.examples.Sequencer,
+    trackingGeometry: acts.TrackingGeometry,
+    field: acts.MagneticFieldProvider,
+    # directNavigation: bool = False,
+    inputProtoTracks: str = "truth_particle_tracks",
+    multipleScattering: bool = False,
+    energyLoss: bool = False,
+    clusters: str = None,
+    calibrator: acts.examples.MeasurementCalibrator = acts.examples.makePassThroughCalibrator(),
+    logLevel: Optional[acts.logging.Level] = None,
+) -> None:
+    customLogLevel = acts.examples.defaultLogging(s, logLevel)
+
+    ## TODO add and test directNavigation
```

Comment:
I thinkd we shouldn't do the broken direct-fit with the sorted prototracks, because this will miss material surfaces which do not carry measurments (like in the ODD)... Rather think about using the `RefittingAlgorithm` lator on
I think you can remove this code...